services/director/src/simcore_service_director/api/v0/openapi.yaml

```diff
@@ -307,6 +307,10 @@ paths:
                                   example:
                                     - Dog
                                     - true
+                                unit:
+                                  title: Unit
+                                  description: 'Units of this input value, if a physical quantity'
+                                  type: string
```

Comment:
it should be nullable no?

Reply:
This was answered and clarified in PR #2154